src/assets/stylesheets/ContextMenu.scss

```diff
@@ -23,7 +23,7 @@
   .context-menu__item {
     margin: 0;
     background-color: inherit;
-    color: inherit;
+    color: var(--rpf-button-secondary-text-color);
```

Comment:
Think this change turns the text the same colour as the background. `color: rpf-button-secondary-color` looks like it might work. I think `inherit` might work on the `border-color`


![image](https://github.com/RaspberryPiFoundation/editor-ui/assets/69108995/8b838b3c-086c-48fd-8da7-32d8ae7966f8)